src/framework/theme/components/cdk/overlay/_overlay.theme.scss

```
@@ -9,3 +9,14 @@
     z-index: 1040;
   }
 }
+
+/**
+ * In case when Nebular Layout custom scroll `withScroll` mode is enabled
+ * we need to disable default CDK scroll blocker (@link NbBlockScrollStrategyAdapter) on HTML element
+ * so that it won't add additional positioning.
+ */
+.nebular-global-scrollblock {
```

I think it has to be:
```suggestion
.nb-global-scrollblock {
```